src/Tribe/API/Timeline.php

```diff
@@ -11,7 +12,31 @@
  */
 class Timeline {
 
-	protected $properties = [];
+	/**
+	 * @var array
+	 */
+	protected $timeline_event_types = [
+		'eventRegistration'     => [
+			'site_option'    => 'timeline_event_registration_id',
+			'headerTemplate' => 'Ticket purchase occurred at {{#formatDate timestamp}}{{/formatDate}} from the Event Tickets HubSpot Integration app',
```

Comment:
Should these be defined in a method and use `__()`?

Reply:
They are the templates that are saved in HubSpot on the creation of each event type. While they could be translated there is no coding to update them in HubSpot if translated in the future or a way then for someone on a multi language site to choose which one they would like. These templates once saved in HubSpot can be edited in the app in HubSpot so if someone would like to change what it says they have the opportunity to do it there.